backend/backend.go

```diff
@@ -72,6 +60,18 @@ type ProverConfig struct {
 	LoggerOut     io.Writer       // defaults to os.Stdout
 }
 
+// NewProverConfig returns a default ProverConfig with given prover options opts
+// applied.
+func NewProverConfig(opts ...ProverOption) (ProverConfig, error) {
+	opt := ProverConfig{LoggerOut: os.Stdout, HintFunctions: hint.GetAll()}
```

Comment:
Maybe also `std.GetHints()`?

Reply:
import cycle hell :) std uses test package to test itself.

Reply:
:(
In that case - I think it is good as is. It will probably require too much refactoring?